packages/vulcan-lib/lib/server/apollo-ssr/renderPage.js

```diff
@@ -146,6 +201,9 @@ const sendToSink = (sink, {
   
   sink.appendToBody(ssrBody);
   sink.appendToHead(head);
+  if (extraScripts) {
```

Comment:
Probably remove `extraScripts` and just transform `head` into a `heads` argument and have it take an array